networking.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: page
+title: Making HTTP Networking Requests in Flutter
+permalink: /networking/
+---
+
+This page describes how to make HTTP networking requests in Flutter. For
+sockets, see [dart:io][dartio].
+
+* TOC Placeholder
+{:toc}
+
+## Using the `http` package
+
+Flutter supports the [http package][http], version `0.11.3+12` or later.
+
+The first step is to declare a dependency by adding an `http...` line under `dependencies` in
```

Comment:
People might be confused what `http...` means. Consider dropping "an `http....`" and just having: "declare a dependency on the `http` package under...."